### ISSUE TITLE
Check why the cloth is pushed away towards the edge and solve it asap

### ISSUE THREAD

**Issue by csingh27** [csingh27sewts/Masterarbeit#62]

**Comment by csingh27** [comment]
Value loss, critic loss, actor loss are all increasing and not decreasing.

**Comment by csingh27** [comment]
Changed actor_loss = critic_value - log_probs from actor_loss = log_probs - critic_value

**Comment by csingh27** [comment]
Actor loss, critic loss, value loss all are increasing.

**Comment by csingh27** [comment]
cloth_corner change action to []*7 from []*3 also taking corner info into account.

**Comment by csingh27sewts** [comment]
Actions might not be defined correctly.
